devices/WS01_RainSensor.js

```diff
@@ -0,0 +1,19 @@
+const exposes = require('../lib/exposes');
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const tz = require('../converters/toZigbee');
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+const e = exposes.presets;
+const ea = exposes.access;
+
+module.exports = [
+    {
+        zigbeeModel: ['WS01'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
+        model: 'WS01', // Vendor model number, look on the device for a model number
+        vendor: 'eWeLink', // Vendor of the device (only used for documentation and startup logging)
```

Comment:
Since the vendor is eWeLink, could you move this to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/ewelink.js ?